python/pyspark/tests.py

```diff
@@ -179,9 +179,12 @@ def test_in_memory_sort(self):
                          list(sorter.sorted(l, key=lambda x: -x, reverse=True)))
 
     def test_external_sort(self):
+        class CustomizedSorter(ExternalSorter):
+            def _next_limit(self):
```

Comment:
Yes, without the mock, it will take a long time to reach memory limit (slowing down tests).